es-app/src/SystemScreenSaver.cpp

```diff
@@ -600,6 +602,8 @@ void SystemScreenSaver::selectGame(bool launch)
 		view->setCursor(gameToSelect);
 		if (launch)
 			view->launch(gameToSelect);
+		else
+			view->setViewportTop(TextListComponent<FileData>::REFRESH_LIST_CURSOR_POS);
```

Comment:
Let's just add a comment to this if branch explaining why we need this, for future reference.